src/js/components/aboutTheData/DataSourcesAndMethodologiesPage.jsx

```diff
@@ -0,0 +1,306 @@
+                                    </p>
+                                    <p>
+                                        You may find the column helpful if you are interested in seeing if the agency’s data has been updated since the last time you viewed it, or if the submission has been certified by the agency. The modal adds a layer of detail which may be helpful if you wish to see all dates on which updates were made to the agency’s data for that period.
+                                    </p>
+                                    <p>
+                                        The table will update for a new period/quarter on the day after that period/quarter submission deadline passes. This timing matches when USAspending.gov reveals the new data across the site. If an agency publishes data ahead of the deadline, the table will show that early publication date once the new period/quarter appears on the table (i.e., on the day after the submission deadline).
+                                    </p>
+                                    <p>
+                                        The submission deadline for the third monthly submission period of a quarter is often earlier than the quarterly submission deadline; thus, quarterly data will be slightly delayed and will be missing if you view the table in the few days between the deadlines. For example, Fiscal Year (FY) 2020 Period 09 data was due on July 30, 2020. Thus, the column would have populated for monthly reporting agencies on July 31. The analogous FY 2020 Q3 data was due on August 14, 2020 and thus, the table would not have populated for quarterly reporting agencies until August 15, 2020. This gap will only exist until FY 2022 at which point the few agencies still reporting on a quarterly basis will transition to reporting monthly.
+                                    </p>
+                                    <p>
+                                        The policy governing which agencies submit quarterly vs. monthly is found in
+                                        <ExternalLink
+                                            url="https://www.whitehouse.gov/wp-content/uploads/2020/04/Implementation-Guidance-for-Supplemental-Funding-Provided-in-Response.pdf">
+                                            OMB’s Memorandum M-20-21 (Appendix A, Section III).
+                                        </ExternalLink>
+                                        The full schedule of deadlines for agency submissions is found on the
+                                        <ExternalLink
+                                            url="https://fiscal.treasury.gov/data-transparency/resources.html">
```

Comment:
This should be a link to a new tab but should not prompt leaving fiscal service since this is a fiscal service website.